src/main/java/org/jenkinsci/plugins/workflow/libs/LibraryStep.java

```diff
@@ -265,16 +263,10 @@ public AutoCompletionCandidates doAutoCompleteIdentifier(@AncestorInPath ItemGro
         @Override public Object invokeMethod(String name, Object _args) {
             Class<?> c = loadClass(prefix + clazz);
             Object[] args = _args instanceof Object[] ? (Object[]) _args : new Object[] {_args}; // TODO why does Groovy not just pass an Object[] to begin with?!
```

Comment:
nit: `InvokerHelper` handles this conversion internally via `InvokerHelper.asArray`, so I think this line could be removed now.